src/views/conference/2018/index/index.jsx

```diff
@@ -0,0 +1,114 @@
+var FormattedDate = require('react-intl').FormattedDate;
+var FormattedHTMLMessage = require('react-intl').FormattedHTMLMessage;
```

Comment:
You still need to require FormattedMessage in addition to FormattedHTMLMessage as both are used.